docs/root/operations/traffic_capture.rst

```diff
@@ -0,0 +1,70 @@
+.. _operations_traffic_capture:
+
+Traffic capture
+===============
+
+Envoy currently provides an experimental :ref:`transport socket extension
```

Comment:
Can you put the experimental status in a warning box? Specifically, I think we should call out the fact that long lived connections are going to cause Envoy to OOM due to proto messages that are too large (phrase however you want). (I would probably also put a giant warning about this in the configuration docs for the file writer.)